searchcore/src/vespa/searchcore/proton/metrics/documentdb_tagged_metrics.cpp

```diff
@@ -213,14 +213,15 @@ DocumentDBTaggedMetrics::MatchingMetrics::RankProfileMetrics::update(const Match
     queryLatency.addValueBatch(stats.queryLatencyAvg(), stats.queryLatencyCount(),
                                stats.queryLatencyMin(), stats.queryLatencyMax());
     if (stats.getNumPartitions() > 0) {
-        if (stats.getNumPartitions() <= partitions.size()) {
-            for (size_t i = 0; i < stats.getNumPartitions(); ++i) {
-                partitions[i]->update(stats.getPartition(i));
-            }
-        } else {
-            vespalib::string msg(vespalib::make_string("Num partitions used '%ld' is larger than number of partitions '%ld' configured.",
-                                             stats.getNumPartitions(), partitions.size()));
-            throw vespalib::IllegalStateException(msg, VESPA_STRLOC);
+        for (size_t i = partition.size(); i < stats.getNumPartitions(); ++i) {
```

Comment:
Should be `partitions`, compilation fails:
```
/source/searchcore/src/vespa/searchcore/proton/metrics/documentdb_tagged_metrics.cpp:216:25: error: `partition` was not declared in this scope; did you mean `std::partition`?
  216 |         for (size_t i = partition.size(); i < stats.getNumPartitions(); ++i) {
```

Reply:
Yes, forgot the last commit.